.github/workflows/unit-test.yml

```
@@ -590,9 +601,42 @@ jobs:
             mkdir -p "$SITE_DIR/unit/history"
             rm -rf "$SITE_DIR/unit/history"/*
             cp -R "$ALLURE_REPORT_DIR/history/." "$SITE_DIR/unit/history/"
+            echo "Allure history 回写完成"
+          else
+            echo "Allure 报告中未生成 history 目录，从 unit-history.json 手动构建 history-trend.json..."
+          fi
+          # 确保 unit/history/history-trend.json 存在（Allure 不生成时手动补充，避免 history 死循环）
+          if [[ ! -f "$SITE_DIR/unit/history/history-trend.json" ]] && [[ -f "$SITE_DIR/unit/unit-history.json" ]]; then
+            mkdir -p "$SITE_DIR/unit/history"
+            python3 -c "
+import json
```

actionlint (1.7.12)

[error] could not parse as YAML: could not find expected ':'

(syntax-check)